packages/attribute-registry-process/src/utilities/errorMappers.ts

```diff
@@ -10,6 +10,7 @@ const {
   HTTP_STATUS_INTERNAL_SERVER_ERROR,
   HTTP_STATUS_FORBIDDEN,
   HTTP_STATUS_NOT_FOUND,
+  HTTP_STATUS_CONFLICT,
```

Comment:
I am improving attribute-registry error mapping so that `409 Conflict` is returned when `"attributeDuplicate"` is thrown. This way, the `409` is bubbled up to the API GW and returned as the GW error with the right code. If attribute-registry is not supposed to return `409` in this case, I can revert the change